internal/include/jemalloc/internal/util.h

```diff
@@ -140,8 +140,8 @@
 #ifdef JEMALLOC_H_EXTERNS
 
 int	buferror(int err, char *buf, size_t buflen);
-uintmax_t	malloc_strtoumax(const char *restrict nptr,
-    char **restrict endptr, int base);
+uintmax_t	malloc_strtoumax(const char* __restrict__ nptr,
```

Comment:
Should we be unsetting JEMALLOC_HAS_RESTRICT in jemalloc_internal_defs.h instead?